### ISSUE TITLE
Clone a vm from a cloned vm with annotationFilters  "!kubevirt.io/*" will make clone stuck as well as clone-controller stuck

### ISSUE THREAD

**Issue by Longchuanzheng** [kubevirt/kubevirt#10387]
**What happened**:
When I want to clone a vm from a vm which is cloned, I set annotationFilters like this:
```
 annotationFilters:
    - "*"
    - "!kubevirt.io/*"
```
the vmclone will stuck in Restoreinprogress phase
![image](https://github.com/kubevirt/kubevirt/assets/49519342/6e7bcb65-1811-454b-ab26-ca29d1dd2e6f)
the virtualmachinerestore has filed 
![image](https://github.com/kubevirt/kubevirt/assets/49519342/bee7923e-2bc9-4697-9ccb-be855fe63e4d)
![image](https://github.com/kubevirt/kubevirt/assets/49519342/cd80bc20-f6c4-4920-a85f-cc6ff7c788bd)

The vm has being created while annotations "kebevirt.io/*"being update by restore-controller also vm-controller.
this will cause controller stuck.

**What you expected to happen**:
Clone vm success no mater what I set in annotationFilters and labelFilters or we need some validating webhook.

**How to reproduce it (as minimally and precisely as possible)**:
clone a vm VM1 from a vm 
clone vm VM2 from VM1 with annotationFilters:
```
 annotationFilters:
    - "*"
    - "!kubevirt.io/*"
```

**Additional context**:

**Environment**:
- KubeVirt version (use `virtctl version`): 0.59
- Kubernetes version (use `kubectl version`): N/A
- VM or VMI specifications: N/A
- Cloud provider or hardware configuration: N/A
- OS (e.g. from /etc/os-release): N/A
- Kernel (e.g. `uname -a`): N/A
- Install tools: N/A
- Others: N/A

**Comment by matthewei** [comment]
I will do the same environment. come back soon

**Comment by iholder101** [comment]
Thank you @Longchuanzheng!
I'm now working on a fix here: https://github.com/kubevirt/kubevirt/pull/10393.

p.s. would you mind adding this BZ to your issue description? https://bugzilla.redhat.com/show_bug.cgi?id=2224663

**Comment by Longchuanzheng** [comment]
> Thank you @Longchuanzheng! I'm now working on a fix here: #10393.
> 
> p.s. would you mind adding this BZ to your issue description? https://bugzilla.redhat.com/show_bug.cgi?id=2224663
I approve

**Comment by iholder101** [comment]
Hey @Longchuanzheng!
There's no need to close the issue :) it will be automatically closed when https://github.com/kubevirt/kubevirt/pull/10393 gets merged.

Thanks a lot for reporting this bug!
/reopen
